src/wp-includes/formatting.php

```diff
@@ -3379,52 +3385,6 @@ function wp_targeted_link_rel_callback( $matches ) {
 	return "<a $link_html>";
 }
 
-/**
- * Adds all filters modifying the rel attribute of targeted links.
- *
- * @since 5.1.0
- */
-function wp_init_targeted_link_rel_filters() {
```

Comment:
To avoid fatal errors in plugins calling the function, this and `wp_remove_targeted_link_rel_filters() will need to be added to deprecated.php too as a noop function.

```php
/**
 * Adds all filters modifying the rel attribute of targeted links.
 *
 * @since 5.1.0
 * @deprecated 6.7.0
 */
function wp_init_targeted_link_rel_filters() {
	_deprecated_function( __FUNCTION__, '6.7.0' );
}